version.json

```diff
@@ -1 +1 @@
-{ "versionName": "2024.1.3", "versionCode": 130 }
+{"versionName":"2024.1.4","versionCode":131}
```

Comment:
why did the formatting change here?

Reply:
This was the auto commit of deliverino. Before was the manuall commit, where some whitespace were added by autoformat. We should stick with this new format deliverino uses.